### PR TITLE
qemu-test-init: fix git repo access and simplify service handling

### PR DESCRIPTION
**Pull request by jluebbe**

### CODE REVIEW

File: qemu-test-init [hunk 1]
```diff
@@ -119,15 +109,29 @@ if type casync; then
...
-if [ -n "$SERVICE" ]; then
-  mkdir /tmp/rauc
-  cp qemu-test-rauc-config /tmp/rauc/system.conf
-  cp test/openssl-ca/dev-ca.pem /tmp/rauc/ca.cert.pem
-  if [ -n "$SERVICE_BACKTRACE" ] && type gdb; then
-    gdb --batch --ex "run" --ex "thread apply all bt" --args rauc service --conf=/tmp/rauc/system.conf &
-  else
-    rauc service --conf=/tmp/rauc/system.conf &
+# rauc system config
+rm -rf /etc/rauc
```
Comment (ejoerns) [inline qemu-test-init:113]: Maybe add a comment why this is needed

File: qemu-test-init [hunk 2]
```diff
@@ -119,15 +109,29 @@ if type casync; then
...
+mkdir /etc/rauc /tmp/rauc
```
Comment (ejoerns) [inline qemu-test-init:114]: Do we still need to create `/tmp/rauc`?

**Review by ejoerns — COMMENTED** [review]
Looks quite useful! :+1: 

Just two minor notes. And If you plan to change anything you might want to replace  the `i.e.` in the last commit message by `e.g.` ;)

File: qemu-test-init (hunk 2 above)
Comment (jluebbe) [reply qemu-test-init:114]: I didn't want to risk changing the behavior in this PR.

**Review by ejoerns — APPROVED** [review]

**Pull request merged**